src/en/guide/introduction/whatsNew.adoc

```diff
@@ -6,4 +6,35 @@ Grails framework 7 updates Spring Boot to version 3.4. We recommend checking the
 
 * https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.4-Release-Notes[Spring Boot 3.4]
 
-The minimum Java version required to run Grails 7 has been updated to Java 17.
\ No newline at end of file
+The minimum Java version required to run Grails 7 has been updated to Java 17.
+
+=== Ubiquitous Containerized Browser Testing with Geb
+
+The https://github.com/grails/geb[Grails Geb Plugin] has received a significant update, introducing test fixtures that enable ubiquitous containerized browser testing.
+
+This new approach is now the recommended way to write functional tests in Grails. However, the previous method using WebDriver binaries remains supported for backward compatibility.
+
+==== Key Features
+
+By extending your test classes with `ContainerGebSpec`, your tests will automatically leverage a containerized browser provided by https://www.testcontainers.org[Testcontainers]. This setup eliminates the need for managing browser versions and ensures consistent test environments.
+
+==== Requirements
+
+To use `ContainerGebSpec`, ensure that you have a compatible container runtime installed. Supported options include:
+
+- **Docker Desktop**
+- **OrbStack** (macOS only)
+- **Rancher Desktop**
+- **Podman Desktop**
+- **Colima** (macOS and Linux)
+
+==== How It Works
+
+Once a compatible container runtime is installed, no additional configuration is needed. Simply extend your test classes with `ContainerGebSpec` (instead of `GebSpec`), and the following will happen:
+
+1. A container will be started automatically when you run your integration tests.
+2. The container will be configured to launch a browser capable of accessing your application under test.
+
```

Comment:
Should it be mentioned, that `GebReportingSpec` is also replaceable by `ContainerGebSpec` with the added `static boolean reportingSpec = true`  (I might remember wrong what the prop name is)

Reply:
It's an annotation to turn on reporting on ContainerGebSpec now.